src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs

```diff
@@ -396,6 +396,8 @@ pandocOverrides = set phaseOverrides postInstall
     postInstall = unlines [ "postInstall = ''"
                           , "  mkdir -p $out/share/man/man1"
                           , "  mv \"man/\"*.1 $out/share/man/man1/"
+                          , "  mkdir -p $out/share/bash-completion/completions"
+                          , "  mv data/bash_completion.tpl $out/share/bash-completion/completions/pandoc"
```

Comment:
Should probably use `pandoc --bash-completion` and thus needs to be tied to a check whether we are cross compiling, that's easier in nixpkgs.